testsuite/MDAnalysisTests/formats/test_libdcd.py

```
@@ -99,3 +99,12 @@ def test_iteration_2(self):
         with self.dcdfile as f:
             for frame in f:
                 pass
+
+class DCDWriteHeaderTest(TestCase):
+
+    def setUp(self):
+        self.dcdfile = DCDFile(DCD)
+
+    @run_in_tempdir()
+    def test_write_header(self):
+        self.dcdfile._write_header()
```

This should raise an error since the dcd isn't opened in write mode